owlapi/src/test/java/org/aksw/owlapi/OwlapiApplicationTests.java

```diff
@@ -0,0 +1,21 @@
+package org.aksw.owlapi;
+
+import org.aksw.owl2nl.OWLAxiomConverter;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class OwlapiApplicationTests {
+
+	@Test
+	void contextLoads() {
+	}
+
+	@Test
+	void Check() throws Exception {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.